package.json

```diff
@@ -1,19 +1,40 @@
 {
   "name": "timberlake",
-  "version": "1.0.0",
+  "version": "0.0.1",
```

Comment:
I think this will break semantic versioning, though I doubt that anyone us using this as a library